src/applications/claims-status/components/claim-files-tab/AddFilesForm.jsx

```diff
@@ -271,7 +282,10 @@ class AddFilesForm extends React.Component {
         <VaButton
           id="submit"
           text="Submit files for review"
-          onClick={this.submit}
+          onClick={() => {
+            clickHandler();
```

Comment:
Most design system components have baked in analytics so this would be sending duplicate events. Add the `disable-analytics` prop if you would like to send your own custom event:

https://design.va.gov/storybook/?path=/docs/components-va-button--primary